lib/install/template.rb

```diff
@@ -37,6 +41,6 @@
 "babel-plugin-transform-object-rest-spread"
 
 puts "Installing dev server for live reloading"
-run "yarn add --dev webpack-dev-server"
+run "#{node_bundler} --dev webpack-dev-server"
```

Comment:
[Same](https://github.com/rails/webpacker/pull/475/files#r120336584) here, `--dev` does not work with `npm`.